ui/src/app/components/webcomponents/action-test-data-runtime-variable-suggestion.component.html

```diff
@@ -71,7 +71,8 @@
     </div>
 
     <!-- List Items-->
-    <div class="list-container h-75">
+    <div class="list-container"
```

Comment:
this can be improved by using ternary operator @manikanta-reddy-t

Reply:
updated according to review comments